db/migrate/20160219143611_create_surveys.rb

```diff
@@ -0,0 +1,9 @@
+class CreateSurveys < ActiveRecord::Migration
+  def change
+    create_table :surveys do |t|
+      t.references :user, null: false
+      t.string :title, null: false, limit: 140
+      t.timestamps null: false
```

Comment:
Maybe here, add a column called `ready` or `active` or `live` or whatever which is a boolean that is set to true when the survey has all of its questions and choices made. Until that time, you wouldn't display or allow people to access and answer the survey. This would also give you ability to prevent the survey creator from making changes or new questions once they've finished the survey. But until that time, they can go back and add more questions etc.

When you click "finish survey" you would switch this to true.